server/chat-plugins/chat-monitor.ts

```diff
@@ -642,13 +642,71 @@ export const commands: ChatCommands = {
 			this.parse(`/join view-filters`);
 		},
 		help(target, room, user) {
-			this.parse(`/help filter`);
+			this.run(`filterhelp`);
+		},
+		test(target, room, user) {
+			this.privatelyCheckCan('lock');
```

Comment:
```suggestion
			this.checkCan('lock');
```

No longer needs to be private now that the individual subcommands aren't aliases.